src/clj/witan/send/send.clj

```diff
@@ -102,12 +102,23 @@
 
 (defn run-model-iteration
   "Takes the model & transitions, transition params, and the projected population and produce the next state of the model & transitions"
-  [simulation {:keys [joiner-beta-params joiner-state-alphas
+  [modify-transitions-from
+   simulation {:keys [joiner-beta-params joiner-state-alphas
                       leaver-beta-params
                       mover-beta-params mover-state-alphas
-                      valid-year-settings] :as params}
+                      valid-year-settings] :as standard-projection}
+   {:keys [modified-joiner-beta-params modified-joiner-state-alphas
+           modified-leaver-beta-params modified-mover-beta-params
+           modified-mover-state-alphas] :as scenario-projection}
    {:keys [model transitions]} [calendar-year projected-population]]
-  (let [cohorts (step/age-population projected-population model)
+  (let [params (if (= 2000 modify-transitions-from)
```

Comment:
Input parameters to the model cannot be optional so it needed to be something, 2000 is just the default, as it is essentially an impossible number that a user would want to I could change project from. It could just as easily be 1900, if that make it more explicit?

Reply:
possibly, but I'd def that as well so that people don't get confused default-start-year or something